blog/_posts/Hola, Amigos.md

```diff
@@ -0,0 +1,118 @@
+# "We solemnly swear we’re up to no good."
+
```

Comment:
Hi @avneet14027  ! Thanks for the PR. Please check the other blog posts for the proper markdown format. We need the whole set of parameters at the start.